### PR TITLE
docs: match 'environments' explanations and sample manifests

### PR DESCRIPTION
**Pull request by huanjani**
As we have added new workload types and new fields to our manifests, our docs have become a little clunky. This is just a quick fix to make the `environments` description for each manifest page match the sample manifest at top, but we should do a more comprehensive fix-up soon.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: site/content/docs/manifest/scheduled-job.en.md [hunk 1]
```diff
@@ -22,6 +22,11 @@ List of all available properties for a `'Scheduled Job'` manifest. To learn abou
       LOG_LEVEL: info
     secrets:
       GITHUB_TOKEN: GITHUB_TOKEN
+      
+    # You can override any of the values defined above by environment.
+    environments:
+      prod:
+        cpu: 2048               # Larger CPU value for prod environment
```
Comment (efekarakus) [inline site/content/docs/manifest/scheduled-job.en.md:29]: nit: for this they will also need to scale the memory as well! 
If we can update to include a mem field as well 🙏

**Review by efekarakus — APPROVED** [review]
Awesome thank you so much!

File: site/content/docs/manifest/scheduled-job.en.md (hunk 1 above)
Comment (huanjani) [reply site/content/docs/manifest/scheduled-job.en.md:29]: Done. 
Should we add a `memory` field to the commented out example in the manifest template? Or link to https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html?

Comment (bvtujo) [reply site/content/docs/manifest/scheduled-job.en.md:29]: I think linking to it in our [`cpu`/`memory`](https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#cpu) docs the way we do is sufficient

**Review by bvtujo — APPROVED** [review]

**Pull request merged**